### ISSUE TITLE
Mention jump button not functioning

### ISSUE THREAD

**Issue by sanderDijkxhoorn** [evelyneee/accord#35]
<img width="487" alt="image" src="https://user-images.githubusercontent.com/15117158/181381411-697586c4-5923-41bd-a430-7f35612afb68.png">
If you press a mention jump button it takes you back to the dm's list.

**Comment by evelyneee** [comment]
I can't seem to reproduce this..

**Comment by sanderDijkxhoorn** [comment]
> ~I can't seem to reproduce this..~ If I use the same message it doesn't work. Guess I'll test with that one message from CS.

<img width="1112" alt="image" src="https://user-images.githubusercontent.com/15117158/181464997-e367a141-6bca-424e-add2-ac184cb406ba.png">
Not sure if this has anything to do with it.

I only have gateway stream compression on in settings.

**Comment by evelyneee** [comment]
Nah, the issue is that the message doesn't attach a guild id. It shouldn't be too big of a deal though.
